karabo_data/tests/mockdata/detectors.py

```diff
@@ -17,22 +17,37 @@ class DetectorModule:
         'trailer',
     ]
 
-    def __init__(self, device_id, frames_per_train=64):
+    def __init__(self, device_id, frames_per_train=64, raw=True):
         self.device_id = device_id
         self.frames_per_train = frames_per_train
+        if not raw:
+            # Drop the first dim (gain) if not raw (proc data)
```

Comment:
This is a bit of a 'repeat the code in English' comment ;-) How about something like:

```python
# Raw data has an extra dimension, used in AGIPD to separate data and gain.
# This dimension is removed by the calibration process.
```

Reply:
Okidoke